src/Microsoft.Identity.Web.TokenAcquisition/MergedOptions.cs

```diff
@@ -49,6 +49,7 @@ public ConfidentialClientApplicationOptions ConfidentialClientApplicationOptions
         public string? RedirectUri { get; set; }
         public bool EnableCacheSynchronization { get; set; }
         internal bool MergedWithCca { get; set; }
+        internal bool PreserveAuthority { get; set; }
```

Comment:
Consider adding some comments and a link to the work item here. We will forget why we ever needed this.